template/src/components/counter.tsx

```diff
@@ -16,12 +16,12 @@ interface DOMIntent {
     link$: Stream<null>;
 }
 
-export function Counter({ DOM, onion }: Sources<State>): Sinks<State> {
+export function Counter({ DOM, state }: Sources<State>): Sinks<State> {
     const { increment$, decrement$, link$ }: DOMIntent = intent(DOM);
 
     return {
-        DOM: view(onion.state$),
-        onion: model(increment$, decrement$),
+        DOM: view(state.stream),
```

Comment:
Do you know why the $ is gone? 

Is the convention around this deprecated?

Reply:
No, but the `$` is kinda an abbreviation for `stream`. So this could be either `sources.state.state$` (pronounced "sources dot state dot state stream") or `sources.state.stream` because `stream$` would mean "stream stream". We thought that `state.state$` also sounds bad, so this is why it's spelled out

Reply:
Ho yeah, it sounds right :)